kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -85,12 +85,16 @@ func NewGitTestingHarness(configPath string) Tester {
 // extracted at all, or the extent to which we got good file coverage from the
 // extraction.
 //
-// TODO(danielmoy): consider better metrics here. For example instead of a
-// single bit representing extraction/download success, consider file count (or
-// if we've done indexing, symbolc ount).
+// TODO(danielmoy): consider better metrics here. For example consider having
+// the smoke test harness try to run a kythe indexer in addition to an
+// extraction and see how much symbol coverage we have.  This might be out of
+// scope for a simple smoke test harness though.
 type Result struct {
 	// Whether the repo was successfully downloaded or extracted.
 	Downloaded, Extracted bool
+	// The number of downloaded and extracted files.
```

Comment:
Are there ever times in which there are 0 downloaded or extracted files but we want to consider the download or extraction to have been successful?

Reply:
Yes, empty repos are technically speaking successes (/futurama pedant meme)